docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,38 @@
+# # Quick start
+
+## What you'll do
+
+- Deploy the charm.
+- Provide GitHub runners on OpenStack mode with images.
+
+## Requirements
+
+- GitHub runner running in OpenStack mode.
+- A running instance of [OpenStack](https://microstack.run/docs/single-node).
+
+## Steps
```

Comment:
I would advise removing the explicit numbering of the steps. To me, it makes the tutorial feel more like a how-to guide.